### DISCUSSION TITLE
Inquiry: About using the vanilla-extract logo

### DISCUSSION THREAD

**Discussion by f4-miyata** [vanilla-extract-css/vanilla-extract#1518] [Q&A] [answered]
Hello. I am the PR representative of a Japanese IT company. 
I would like to use the vanilla-extract logo in our technology stack slide, but I couldn't find any official logo data and guidelines on GitHub, so I'm not sure if it's okay to use. 
Does anyone have any documents or have you contacted them about this? 
I apologize for the non-technical question, but any assistance you could provide would be greatly appreciated.

**Comment by askoufis** [comment, marked as answer]
I ended up answering your question [in discord](https://discord.com/channels/885877446098964512/885877446891667529/1321386631781027882) where you asked it first, but I'm guessing you didn't see it before creating this discussion 😅. Regardless, I think it's worth copying my response (with a bit more added detail) here so it's easier to find in the future.

Discord response:
> We don't really have an official logo. It's really just the cupcake emoji. We have an SVG version of a specific emoji that we liked and chose to be the logo on the website in the repo https://github.com/vanilla-extract-css/vanilla-extract/blob/9e5438df05ebbe06e78007ca469a4d37d3d1a092/site/logo.svg.

You're more than welcome to use that in your slides, though if it's easier to just use the cupcake emoji then that's perfectly fine too.

With a bit more digging, I found that the cupcake we use is specifically the one available in the Google Noto Color Emoji fonts that came with Android versions [9.0](https://emojipedia.org/google/android-9.0/cupcake) to 11.0. The cupcake changed in v12.

**Reply by f4-miyata** [reply]
Thank you so much for the detailed response! 
It seems there was a bit of a mix-up on our end, and I ended up asking the same question on Discord as well. My apologies for the duplication.

I’ll definitely use the logo or the emoji in my slides, and I’ll make sure to credit the project appropriately. Thanks again for taking the time to explain this so thoroughly!
